samples/sample/app-browser/build.gradle.kts

```diff
@@ -16,12 +16,8 @@ val KERMIT_VERSION: String by project
 
 kotlin {
     js {
-        browser {
-            runTask {
-                outputFileName = "app-browser.js"
```

Comment:
Does this no longer need to be specified anywhere?

Reply:
yeah it takes the name of the module by default which is `app-browser` already